frontend/src/metabase/components/SaveQuestionForm/context.tsx

```diff
@@ -30,6 +30,23 @@ type SaveQuestionContextType = {
 export const SaveQuestionContext =
   createContext<SaveQuestionContextType | null>(null);
 
+/*
+ * Why are we using these useState calls?
+ *
+ * When we use SaveQuestionModal within the QueryModals, the 'opened' prop on the modal
```

Comment:
I see opened={true} on every usage though, not only on QueryModals?

Reply:
Just trying to give context for why we had to do this in the first place

Reply:
I see, I thought this was related to the PR i found in git blame where you split the files and (it think?) introduced the context, but I see all the things that confuse me were there even before that 😅

Reply:
ah yeah the context was never the problem 😅